dclab/__init__.py

```diff
@@ -15,6 +15,8 @@
 from .rtdc_dataset import new_dataset
 from .rtdc_dataset.feat_temp import register_temporary_feature, \
```

Comment:
I guess there is no difference between how to do line continuations for imports, but PEP328 says to use parentheses (and I like it :) ) https://www.python.org/dev/peps/pep-0328/
If you agree then I can go through the code and do this everywhere. Or we can get @maxschloegel to write that python formatting package we mentioned ;)